removeVault.py

```diff
@@ -31,7 +34,10 @@ def process_archive(archive_list):
 
 				logging.info('Retry to remove archive ID : %s', archive['ArchiveId'])
 				try:
-					vault.delete_archive(archive['ArchiveId'])
+					glacier.delete_archive(
+					    vaultName='vaultName',
```

Comment:
Ahah! Yopu're right! I had to fix this while testing on my EC2 and fogot to apply the fix in the git repo. Thanks!